GitExtensions/Program.cs

```diff
@@ -116,12 +116,12 @@ private static void RunApplication()
                 formChoose.ShowDialog();
             }
 
-            if (!AppSettings.TelemetryEnabled.HasValue)
```

Comment:
Is the change really the same, many properties cannot be handled like this

Reply:
I believe this has identical behaviour.

https://sharplab.io/#v2:EYLgHgbALANALiATgVwHYB8ACAmAjAWACgjNcBOACgCIBnAYwFNUBDRASwHsACXKgSgDcRAMIA6ALK4KgoiXLV6TVpy7Z+QwmPHZpGuRC44uwogG8iXS4YDMh3AeAcOAGwD8XAAqIOABwtXzQitgrgBzBjguUztKKnC4OAZEdUMAdi5UZGdnAS4AX38QrhoIqJiFCMTkwXzC2qCrTFtSA0woLklpOsCiyy9fLldXAF4uOBQGDWCChssmu1b27S7ZqLrg/p9BkbGJqasZvKA=

The generated code is identical too.

https://sharplab.io/#v2:EYLgHgbALANALiAlgGwD4AEBMBGAsAKAIHoAqA9bATgAoAiAZwGMBTAOwEMAnRAewAJstAJQBuAgGEAdAFls1UQXJU6TNl159MwsfinTM8nSSKL8FCHyx9xBAN4E+jywGZL2C8B49kAfj4AFTh4ABwcne3wnKL4Ac2Y4Pls3Glo4uDhmTm1LAHY+VgBXZGQRPgBfMOi+enjE5JV4jKzRcsrWyKd0V3NLKD5ZeTaIqsdAkL4fHwBePjhOAuYdKIqOxy63C3Q+/UHVxLaoseCJ6dn5xbaVsqA=